src/condition.cpp

```diff
@@ -1567,6 +1567,10 @@ static std::function<T( const dialogue & )> get_get_translation_( const JsonObje
         return [ma, ret_func]( const dialogue & d ) {
             return ret_func( matec_id( ma.evaluate( d ) )->name );
         };
+    } else if( jo.get_string( "mutator" ) == "topic_item" ) {
+        return []( const dialogue & d ) {
+            return d.cur_item.str();
```

Comment:
Do you want to return the ID of the item? or it's in game name? If you want the in game name you need to do something like "cur_item->nname(1);" and if you want the ID this should be moved above to get_str from get_translation

Reply:
I want ID, THX!